portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -14,19 +14,38 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+/** Servlet that stores comments. */
+@WebServlet("/comments")
+public class CommentsServlet extends HttpServlet {
 
+  private List<String> comments;
+
+  /* Initializes comments list with hard-coded data for now. */
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+    comments.add("Hello");
+    comments.add("World");
+  }
+
+  /* Returns comments as a JSON string. */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Convert comments to JSON using Gson
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
```

Comment:
better naming? like commentsInJsonFormat?

Reply:
Ok I've renamed the variable.